src/test/scala/com/gd/twitteranalytics/TweetsTransformerTest.scala

```diff
@@ -0,0 +1,59 @@
+package twitteranalytics
+
+import java.sql.Timestamp
+import java.util.Date
+
+import com.gd.twitteranalytics.TweetsTransformer
+import com.holdenkarau.spark.testing.{DataFrameSuiteBase, StreamingSuiteBase}
+import org.scalatest.FreeSpec
+import twitter4j.{TwitterObjectFactory}
+import org.scalamock.scalatest.MockFactory
+
+import scala.io.Source
+
+class TweetsTransformerTest extends FreeSpec with StreamingSuiteBase with MockFactory  with DataFrameSuiteBase {
+
+  "TransformTweets" - {
+    val constDate = new Date("Wed Sep 26 10:37:20 PDT 2018")
+    val text = "This is the #Sample #text for #test purpose"
+    val origDate = new Timestamp(constDate.getTime)
+
+    "getDateAndText function" - {
+    "when presented with a complete tweet status" - {
+      "should retrieve date and text from the tweets status" in {
+        val rawJson = Source.fromURL(getClass.getResource("/tweetStatus.json")).getLines.mkString
+        val tweetStatus = TwitterObjectFactory.createStatus(rawJson)
+        val input = List(List(tweetStatus))
+        val expected = List(List((constDate,text)))
```

Comment:
could you clarify please, does  `const...` state for "constant"? 

NOTE: the code should describe domain and/or use-cases, not just state an implementation fact.

Reply:
NOTE: I would not ask the above question if you would not shorten the variable name. The shortened version may be clear to you, but not to reviewer.